db/re/skill_db.yml

```diff
@@ -21913,7 +21911,7 @@ Body:
       Skill:
         Reproduce: true
     Duration1: 15000
-    Duration2: 15000
+    Duration2: 13000
```

Comment:
Maybe suspicious.

Reply:
I confirmed it to be 13 seconds on Aegis.